lib/modules/ens/index.js

```diff
@@ -34,6 +35,9 @@ class ENS {
   }
 
   setProviderAndRegisterDomains(cb) {
```

Comment:
Can't comment on the right line, but maybe just changing ` this.embark.registerActionForEvent("contracts:deploy:afterAll", this.setProviderAndRegisterDomains.bind(this));` to ` this.embark.once("contracts:deploy:afterAll", this.setProviderAndRegisterDomains.bind(this));` might work (probly need to change the even name, but I'm pretty sure it exists)

Reply:
way much simpler thx @jrainville